client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -43,6 +41,50 @@ import UpgradeNudge from 'my-sites/upgrade-nudge';
 // Utility function for checking the state of the Payment Buttons list
 const isEmptyArray = a => Array.isArray( a ) && a.length === 0;
 
+// Selector to get the form values, insert the missing field, and convert it to
+// a custom post data structure ready to be passed to `wpcom` API.
+const getPostFromForm = state => {
```

Comment:
I think it's not a very clear name. Does it return the product from a form? Either way, using "from" and "form" that close together is hard to read. :)

I would call it `formProductToCustomPost`